src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +141,12 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Sorts the persons in the list in alphabetical order
+     */
+    public void sorts(){
+        Collections.sort(internalList);
```

Comment:
Try not to modify the original data, instead, modify a copy (: This gives more freedom to keep track of some properties of the original `UniquePersonList` and implement more sort commands in the future - e.g. sort by recently added